### PR TITLE
mathcing capabilities: fix list of well-known systems

### PR DESCRIPTION
**Pull request by andreastt**
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/1077)

***
<a href="https://pr-preview.s3.amazonaws.com/andreastt/webdriver/pull/1077.html" title="Last updated on Dec 20, 2017, 2:04 PM GMT (d9ae35b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webdriver/1077/6dc0341...andreastt:d9ae35b.html" title="Last updated on Dec 20, 2017, 2:04 PM GMT (d9ae35b)">Diff</a>

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -2438,12 +2438,12 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
...
          <tr>
-          <th>Platform
-          <th>Operating System
+          <th>Key
+          <th>System
          </tr>
-         <tr><td>linux</td><td>Any OS based upon the Linux kernel and X Windows.</td></tr>
-         <tr><td>mac</td><td>Any version of Apple's OS X.</td></tr>
-         <tr><td>windows</td><td>Any version of Microsoft's Windows, including desktop and mobile versions.</td></tr>
+         <tr><td>"<code>linux</code>"</td> <td>Any system based upon the Linux kernel.</td></tr>
```
Comment (shs96c) [inline webdriver-spec.html]: Android doesn't fit into this category, but is based on the Linux kernel, which is why we had the qualifier about X Windows.

Comment (andreastt) [reply webdriver-spec.html]: Not all Linux systems use X.

Comment (shs96c) [reply webdriver-spec.html]: We need to differentiate Android from Linux. "Any server or desktop system based upon the linux kernel"?

Comment (andreastt) [reply webdriver-spec.html]: Makes sense, fixed.

**Pull request merged**